test/globalErr.js

```diff
@@ -108,4 +108,112 @@ describe('globalErr', function () {
       }, 'Invalid val')
     });
   });
+
+  describe('skipStackTest', function () {
+    // Skip tests if `Error.captureStackTrace` is unsupported
+    if (typeof Error.captureStackTrace === 'undefined') return;
+
+    try {
+      throw Error();
+    } catch (err) {
+      // Skip tests if `err.stack` is unsupported
+      if (typeof err.stack === 'undefined') return;
+    }
+
+    // Note: `.to.not.throw` isn't used for the assertions that are expected to
+    // not throw an error because it'll pollute the very same stack trace which
+    // is being asserted on. Instead, if a test is failing, then Mocha will pick
+    // up the error that's thrown by `err`.
+    describe('falsey', function () {
+      // The inner `err` executes `myGetter` and catches the thrown error. But
```

Comment:
I don't think we need these comments.

They are really well written, but I think it is very easy to understand what is happening just by reading the code. If we have too many comments on the code it just makes it more difficult to maintain because we gotta make sure we update all the related comments, otherwise they will tell a different story than the code does.

IMO the outer code is okay, but this one (inner one) seems too much.

Anyway, I won't reject this PR because of this, please let me know what you think about it. 😄